eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/json/EclairModels.scala

```diff
@@ -0,0 +1,188 @@
+package org.bitcoins.eclair.rpc.json
+
+import play.api.libs.json.{ JsArray, JsObject }
+
+sealed abstract class EclairModels
```

Comment:
Is there a reason that this isn't extended by anything in here? Also is there a difference between `abstract class` and `trait` in this instance?

Reply:
I default to `abstract class` over `trait` because it is "more native" to the JVM.